### PR TITLE
fix: update cluster labels doc to make it clear that labels are not resource tag

### PR DESCRIPTION
**Pull request by andreaangiolillo**
## Description

Please include a summary of the fix/feature/change, including any relevant motivation and context.

Link to any related issue(s): #1370

## Type of change:

- [x] Bug fix (non-breaking change which fixes an issue). Please, add the "bug" label to the PR.
- [ ] New feature (non-breaking change which adds functionality). Please, add the "enhancement" label to the PR.
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected). Please, add the "breaking change" label to the PR.
- [ ] This change requires a documentation update
- [ ] Documentation fix/enhancement

## Required Checklist:

- [x] I have signed the [MongoDB CLA](https://www.mongodb.com/legal/contributor-agreement)
- [x] I have read the [contribution guidelines](https://github.com/mongodb/terraform-provider-mongodbatlas/blob/master/CONTRIBUTING.md)
- [x] I have added tests that prove my fix is effective or that my feature works per HashiCorp requirements
- [x] I have added any necessary documentation (if appropriate)
- [x] I have run make fmt and formatted my code

### CODE REVIEW

File: website/docs/d/advanced_cluster.html.markdown [hunk 1]
```diff
@@ -76,11 +76,14 @@ Specifies BI Connector for Atlas configuration.
...
 ### labels
 
-Key-value pairs that tag and categorize the cluster. Each key and value has a maximum length of 255 characters.  You cannot set the key `Infrastructure Tool`, it is used for internal purposes to track aggregate usage.
+Key-value pairs that categorize the cluster. Each key and value has a maximum length of 255 characters.  You cannot set the key `Infrastructure Tool`, it is used for internal purposes to track aggregate usage.
 
 * `key` - The key that you want to write.
 * `value` - The value that you want to write.
 
+-> **NOTE:** MongoDB Atlas doesn't display your labels.
+-> **NOTE:** Cluster labels are the same as [resource TAGs](https://www.mongodb.com/docs/atlas/tags/). We plan to add [resource TAGs](https://www.mongodb.com/docs/atlas/tags/) support in a future release.
```
Comment (gssbzn) [inline website/docs/d/advanced_cluster.html.markdown]: are or are not?

Comment (andreaangiolillo) [reply website/docs/d/advanced_cluster.html.markdown]: ahaha `are not `.  Thanks!

Comment (andreaangiolillo) [reply website/docs/d/advanced_cluster.html.markdown]: Updated

**Review by gssbzn — APPROVED** [review]
thanks

**Pull request merged**